product_filters/models/product.py

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
+# pylint: disable=expression-not-assigned
```

Comment:
I enabled again and test it and I saw the following output:
```bash
************* Module product_filters.models.product
product_filters/models/product.py:152: [W0106(expression-not-assigned), ProductCategory._get_attributes_related] Expression "(None in i[1]) and (attr_ids2.append(i[0]))" is assigned to nothing
************* Module website_product_filters.controllers.main
website_product_filters/controllers/main.py:102: [W0106(expression-not-assigned), WebsiteSale.shop] Expression "(post.get('product_sorter', '0') != '0') and (res.qcontext.update({'sortby': sortby}))" is assigned to nothing
```

```python
        for i in self._cr.fetchall():
            attr_ids.append(i[0])
            None in i[1] and attr_ids2.append(i[0])
```

The following sentence is assigned to nothing:
`None in i[1] and attr_ids2.append(i[0])`

Maybe, you could fix it with:
```python
if i[1] is None:
   attr_ids2.append(i[0])
```

Or using `# pylint: disable=expression-not-assigned` in the line of code to avoid disable it in the full file